cms/functions.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/* Doc Block
+ * Returns all project titles from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project titles in the arrays within the arrays pulled from the database.
+ */
+function returnProjectTitles(array $array):string {
+    $titles = '';
+    foreach ($array as $value) {
+        $titles .=$value['projectTitle']."\n";
+    }
+    return $titles;
+}
+
+/* Doc Block
+ * Returns all project title links from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project title links in the arrays within the arrays pulled from the database.
+ */
+function returnProjectTitleLinks(array $array):string {
+    $titlesLinks = '';
+    foreach ($array as $value) {
+        $titlesLinks .= $value['projectTitleLink']."\n";
+    }
+    return $titlesLinks;
+}
+
+/* Doc Block
+ * Returns all project image links from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project image links in the arrays within the arrays pulled from the database.
+ */
+function returnProjectImages(array $array):string {
+    $projectImages = '';
+    foreach ($array as $value) {
+        $projectImages .= $value['projectImage']."\n";
+    }
+    return $projectImages;
+}
+
+/* Doc Block
+ * Returns project id's from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of id's or keys in the arrays within the arrays pulled from the database.
+ */
+function returnProjectIds(array $array):string {
+    $projectIds = '';
+    foreach ($array as $value) {
+        $projectIds .= $value["id"]."\n";
+    }
+    return $projectIds;
+}
+
+/* Doc Block
+ * Returns a single project title from array content provided by database.
+ *
+ * @param $array array associative array provided by database query.
+ * @param $recId int number pointing to the record id number in the database.
+ *
+ * @return string the title content for a single record.
+ */
+function returnProjectTitle(array $array, int $recId):string { // change title to record
+    $index = $recId - 1;
+    $projectTitleArray = $array[$index];
+    $projectTitle = '';
+    $projectTitle .=$projectTitleArray['projectTitle'];
```

Comment:
just return $projectTitleArray['projectTitle']; dont need the other lines

Reply:
Edited for all 4 functions that use this.